.github/CODEOWNERS

```diff
@@ -1469,3 +1469,6 @@
 /**/tsconfig.json @witemple-msft @deyaaeldeen
 /**/tsconfig.strict.json @deyaaeldeen
 /.scripts/ @mikeharder @ckairen
+
+# Add owners for notifications for specific pipelines
+/eng/pipelines/aggregate-reports.yml         @xirzec @jeremymeng
```

Comment:
Looks like you also need to remove this from line 20 above as well.

Reply:
@weshaggard done